first_problem.rb

```diff
@@ -0,0 +1,23 @@
+class FirstProblem
+
+  def initialize(input_array)
+    @numbers_array = input_array
+  end
+
+  def compute()
+    if @numbers_array.empty?
+    then
+      return 0
+    end
+
+    sum = 0
+    @numbers_array.each do |number|
+      if number % 2 == 0
```

Comment:
Ruby is quite a nice language. Given that numbers are objects, they respond to '.even?' or '.odd?'
Can you write this first problem in one line?

Reply:
I can sure try ! :)

Reply:
not sure this is what you wanted...